modin/data_management/partitioning/axis_partition.py

```diff
@@ -155,6 +156,13 @@ def split_result_of_axis_func_pandas(axis, num_splits, result):
     Returns:
         A list of Pandas DataFrames.
     """
+    if length_list is not None:
+        length_list.insert(0, 0)
```

Comment:
Why is it padded by two zeros in the beginning? Won't it make the return list's zeroth index be empty?

Reply:
Nevermind I accidently ran my jupyter notebook cell twice.